### PR TITLE
Adds a pure GRU model with variable number of layers

### PR DESCRIPTION
**Pull request by Demborg**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -28,6 +28,30 @@ def forward(self, sequence):
...
+        self.output_layer = nn.Linear(self.hidden_size, 4)
+        self.h0 = nn.Parameter(torch.randn(self.num_layers, 1, self.hidden_size))
+
+    def forward(self, sequence):
+        output, hn = self.gru(sequence, self.h0)
+        predictions = self.output_layer(output)
+        return predictions
+
+    def get_name(self):
+        return "PureGRU_h:{}_l:{}".format(self.hidden_size, self.num_layers)
```
Comment (netrome) [inline models.py]: Do we really want to have ':' in the model names?

File: models.py [hunk 2]
```diff
@@ -28,6 +28,30 @@ def forward(self, sequence):
...
 
+class PureGRU(nn.Module):
+
+    def __init__(self, **kwargs):
+        """
+        Initialize new PureGRU model.
+        :keyword arguments:
+        hidden_size: int, number of hitten units.
+        num_layers: int, Number of recurrent layers
+        """
```
Comment (netrome) [inline models.py:39]: Nice docstring :)

**Review by netrome — COMMENTED** [review]
Nice :)

**Comment by netrome** [comment]
I think this looks good and can be merged.

File: models.py (hunk 1 above)
Comment (Demborg) [reply models.py]: Hmm, should be ok and makes it quite readable/easy to parse automatically but something like - might be better?

**Pull request merged**